### PR TITLE
Rails 6.0

### PR DESCRIPTION
**Pull request by NickLaMuro**
Links
-----

* Epic:  https://github.com/ManageIQ/manageiq/issues/19977
* Current Cross Repo: https://github.com/ManageIQ/manageiq-cross_repo-tests/pull/179

### CODE REVIEW

File: vmware_web_service.gemspec [hunk 1]
```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = "> 2.4"
   spec.files = Dir["{app,config,lib}/**/*"]
 
-  spec.add_dependency "activesupport",        "~> 5.2.4", ">= 5.2.4.3"
```
Comment (bdunne) [inline vmware_web_service.gemspec:18]: Can we apply https://github.com/ManageIQ/activerecord-id_regions/pull/18/files#r537735807 here as well?

**Review by Fryguy — APPROVED** [review]

**Comment by bdunne** [comment]
Is this ready to un-wip?

**Comment by NickLaMuro** [comment]
@bdunne I have some failures in the cross-repo tests that I want to resolve before I commit to this.

**Review by bdunne — APPROVED** [review]

**Comment by bdunne** [comment]
@agrare Can you push a new release to rubygems.org?  I'm not an owner on this one.

**Comment by agrare** [comment]
Released v2.0.3 with this change

**Pull request merged**